### DISCUSSION TITLE
Release v0.3.1

### DISCUSSION THREAD

**Discussion by olback** [olback/library-loader#77] [Releases]
* Added DesignSpark (#75)

<hr /><em>This discussion was created from the release <a href='https://github.com/olback/library-loader/releases/tag/0.3.1'>Release v0.3.1</a>.</em>

**Comment by throbscottle** [comment]
Thanks for adding the DS support. Unfortunately, I get 
```
$ library-loader-cli 
Using config at "/home/name/.config/LibraryLoader.toml"
Started watching "/home/name/circuits/DesignSpark_models"
Active formats:
        designspark => "/home/name/circuits/DesignSpark_models/Library"
Stop by pressing <Enter>
Detected "/home/name/circuits/DesignSpark_models/LIB_74HC00S14-13.zip"
ZipArchiveEmpty
```

I tried with several different models direct from SamacSys, all with the same result :(...

**Comment by BenViete** [comment]
Hi, first thanks for your work on this!

When I run the macOS-compile.sh script, then I get an error message:

> The following warnings were emitted during compilation:
> 
> warning: `"pkg-config" "--libs" "--cflags" "gdk-pixbuf-2.0" "gdk-pixbuf-2.0 >= 2.32"` did not exit successfully: exit status: 1
> 
> error: failed to run custom build command for `gdk-pixbuf-sys v0.14.0`
> 
> Caused by:
>   process didn't exit successfully: `/Users/vieteben/Downloads/library-loader-master/target/release/build/gdk-pixbuf-sys-087ed2d25f7e9e15/build-script-build` (exit status: 1)
>   --- stdout
>   cargo:rerun-if-env-changed=GDK_PIXBUF_2.0_NO_PKG_CONFIG
>   cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-apple-darwin
>   cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_apple_darwin
>   cargo:rerun-if-env-changed=HOST_PKG_CONFIG
>   cargo:rerun-if-env-changed=PKG_CONFIG
>   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
>   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
>   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
>   cargo:rerun-if-env-changed=PKG_CONFIG_PATH
>   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
>   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
>   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
>   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
>   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
>   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
>   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
>   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
>   cargo:warning=`"pkg-config" "--libs" "--cflags" "gdk-pixbuf-2.0" "gdk-pixbuf-2.0 >= 2.32"` did not exit successfully: exit status: 1
>   --- stderr
>   Package libjpeg was not found in the pkg-config search path.
>   Perhaps you should add the directory containing `libjpeg.pc'
>   to the PKG_CONFIG_PATH environment variable
>   Package 'libjpeg', required by 'gdk-pixbuf-2.0', not found

What can I do to manage this?
